packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -119,12 +120,17 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
         // Build an object which contains the key, the count and the display name.
         for (const location of locationsResult) {
             const keys = Object.keys(location.properties.categories);
+
             for (const key of keys) {
+                // Get the categories from the App Config and find the ones with a matching key.
+                const appConfigCategories = appConfigResult?.menuInfo.mainmenu;
+                const appConfigCategory = appConfigCategories.find(category => category.categoryKey === key);
```

Comment:
I don't think it's necessary to have the `appConfigCategories` const. We could just use `appConfigResult?.menuInfo.mainmenu` on this line.

Reply:
I agree 👍